src/psyclone/configuration.py

```diff
@@ -692,9 +693,17 @@ def get_default_keys(self):
         '''
         return self._config.defaults()
 
+    def get_constants(self):
+        ''':returns: the constants instance of the current API.
+        :rtype: either :py:class:`psyclone.domain.lfric.LFRicConstants`, \
+            :py:class:`psyclone.domain.gocean.GOceanConstants`, or \
+            :py:class:`psyclone.domain.nemo.NemoConstants`
+            '''
```

Comment:
Please un-indent the closing ''' and remove the following blank line.

Reply:
Done (my pylint/pycodestyle did not complain about this *sigh*)